databusclient/api/delete.py

```
@@ -0,0 +1,188 @@
+    """
+    Delete a list of Databus resources.
+
+    Parameters:
+    - databusURIs: List of full databus URIs of the resources to delete
+    - databus_key: Databus API key to authenticate the deletion requests
+    """
+    for databusURI in databusURIs:
+        _delete_resource(databusURI, databus_key, dry_run=dry_run, force=force)
+
+
+def _delete_artifact(databusURI: str, databus_key: str, dry_run: bool = False, force: bool = False):
```

Ruff (0.14.7)

Unused function argument: `force`

(ARG001)